packages/faustwp-core/src/components/WordPressTemplate.tsx

```diff
@@ -21,13 +21,6 @@ export type FaustTemplateProps<Data, Props = Record<string, never>> = Props & {
   __TEMPLATE_VARIABLES__?: { [key: string]: any };
 };
 
-function cleanTemplate(
```

Comment:
This is problematic as HMR does not pick up the React Component when it changes.